src/Navigation/Home/index.css

```diff
@@ -0,0 +1,168 @@
+.home {
+  padding: var(--section-padding);
+  font-family: var(--project-font-family);
+}
+.home_part1 {
+  display: flex;
+  flex-direction: row;
+}
+.home_part2 {
+  display: flex;
+  flex-direction: row;
+  padding-top: 60px;
+}
+.profile--logo {
+  background-image: url("../../images/logo.png");
+  width: 50px;
+  height: 50px;
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+.profile--heading1 {
+  color: var(--primary-color);
+  font-size: 48px;
+}
+.profile--heading2 {
+  color: #d4d4d4;
```

Comment:
can this color be made into a variable?
Advantage would be that it will be easier to change it from a single file if we wish to do a different theme.